scene/main/canvas_item.h

```diff
@@ -341,6 +344,10 @@ class CanvasItem : public Node {
 	void draw_end_animation();
 
 	static CanvasItem *get_current_item_drawn();
+	void get_arc_points(Vector<Point2> &r_points, const Vector2 &p_center, real_t p_major, real_t p_minor, real_t p_start_angle, real_t p_end_angle, int p_point_count);
```

Comment:
```suggestion
	void get_arc_points(LocalVector<Point2> &r_points, const Vector2 &p_center, real_t p_major, real_t p_minor, real_t p_start_angle, real_t p_end_angle, int p_point_count);
```
If COW functionality isn't needed